### ISSUE TITLE
Results difference after converting a tensorflow op to TfLite

### ISSUE THREAD

**Issue by MonicaGu** [tensorflow/tensorflow#36653]
**System information**
- Have I written custom code: No
- OS Platform and Distribution: MacOS 10.15
- TensorFlow version: 2.0
- Python version: 3.6.9

**Describe the current behavior**
I converted a Tensorflow model with only one Tensorflow Keras layer _tf.keras.layers.Conv2D_ to TFLite and ran the TFLite model. There was no quantization when converting the model.

**Describe the expected behavior**
The expected output of the TFLite model should be the same as the output of the Tensorflow model. However, the output of the TFLite model is slightly different from the result of the original Tensorflow model. I'm not sure whether this is a bug of TFLite framework or there was something wrong when I converted the Tensorflow model toTFLite.

**Code to reproduce the issue**
```Python
import tensorflow as tf

input = tf.keras.layers.Input(shape=(224, 224, 3), dtype='float32')
output = (filters=64, kernel_size=(7,7), strides=(2,2), padding='valid', data_format="channels_last", activation='linear', dilation_rate=(1,1), bias_initializer='zeros')(input)
tfmodel = tf.keras.Model(inputs=input, outputs=output)

x = tf.random.normal([1,224, 224, 3], dtype=tf.float32)
tf_y = tfmodel(x)

conv = tf.lite.TFLiteConverter.from_keras_model(tfmodel)
lite_bytes = conv.convert()
interp = tf.lite.Interpreter(model_content=lite_bytes)

input_index = interp.get_input_details()[0]['index']
output_index = interp.get_output_details()[0]['index']
interp.resize_tensor_input(input_index, [1, 224, 224, 3])
interp.allocate_tensors()
interp.set_tensor(input_index, x)
interp.invoke()
lite_y = interp.get_tensor(output_index)
```

**Other info / logs**
The issue is similar when the Conv2D layer is replaced with BatchNormalization layer.

**Comment by jvishnuvardhan** [comment]
@MonicaGu I ran your code and find very very small mismatch (in the 7th decimal) in the results between keras  and tflite_model. Do you see significantly different results. Please check the [gist here]( https://colab.sandbox.google.com/gist/jvishnuvardhan/e7d0429feef15a434a36e5226c10f214/untitled819.ipynb). Thanks!

**Comment by MonicaGu** [comment]
@jvishnuvardhan In my case, I also found very small mismatch in the 7th or 6th decimal. I am not sure is this small mismatch inevitable? Or is this caused by a bug in the computation of Conv2D in TFLite? Thank you!

**Comment by terryheo** [comment]
It's expected behavior. Since floating point has limited precision, calculated results can be vary by order of execution. 
https://en.wikipedia.org/wiki/Floating-point_arithmetic#Accuracy_problems
You can see we're allowing 1e-5 (0.00001) errors in our TFLite kernel testing.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/conv_test.cc#L819
So the 7th or 6th decimal difference looks good.
